lib/java_buildpack/jre/open_jdk_like.rb

```diff
@@ -38,10 +38,8 @@ def command
       # (see JavaBuildpack::Component::ModularComponent#sub_components)
       def sub_components(context)
         [
-          JvmkillAgent.new(sub_configuration_context(context, 'jvmkill_agent')),
```

Comment:
We can't remove this here as it'll break other things. Your class ZingJRE inherits from this class though, so what you need to do is to override this method, `sub_components` in your class and supply a list that does not include JVM Kill agent.